pkg/internal/codegen/parse/util.go

```diff
@@ -231,6 +231,17 @@ func hasSingular(t *types.Type) bool {
 	return false
 }
 
+// hasRequired returns true if t is annotated with
+// +required
+func hasRequired(t *types.Type) bool {
+	for _, c := range t.CommentLines {
```

Comment:
why c?

Reply:
c for commentLine, just following the example of all the other util functions in this file